### PR TITLE
Update mkdocs 1.x

### PR DESCRIPTION
**Pull request by KazuCocoa**
Update from 0.17 to 1.0.
Dockerfile, cinder/nav-sub.html and cinder/nav.html are manually changed.

---

https://github.com/appium/appium.io/issues/225

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ To locally preview appium.io
 1. Node + NPM
 1. Ruby Bundler (`gem install bundler`)
 1. pip (`brew install pip`)
-1. MkDocs 0.16.3 (`pip install mkdocs==0.17.5`)
+1. MkDocs 0.16.3 (`pip install mkdocs==1.0.4`)
```
Comment (mykola-mokhnach) [inline README.md:18]: should version numbers be equal?

File: docs/cn/sitemap.xml [hunk 2]
```diff
@@ -1,198 +1,188 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
-
-    
-        
     <url>
-     <loc>/about-appium/intro/</loc>
+     <loc>None</loc>
```
Comment (mykola-mokhnach) [inline docs/cn/sitemap.xml:4]: should it be none?

File: README.md (hunk 1 above)
Comment (KazuCocoa) [reply README.md:18]: Yes, still. I haven't checked over 1.1 (probably it needs to coordinate more)

File: docs/cn/sitemap.xml (hunk 2 above)
Comment (KazuCocoa) [reply docs/cn/sitemap.xml:4]: mm, probably something was missed from 0.17 to 1.0. Will take a look

**Comment by KazuCocoa** [comment]
will work in https://github.com/appium/appium.io/pull/318

**Pull request closed without merging**